python/xgbserver/setup.py

```diff
@@ -33,6 +33,7 @@
     python_requires='>3.4',
     packages=find_packages("xgbserver"),
     install_requires=[
+        "numpy>=1.0"
```

Comment:
kfserving package already include the numpy dependency

Reply:
Updated in new commit